docs/native/src/scenarios/from_app.rst

```diff
@@ -9,42 +9,45 @@ Opening files from your app in |Office iOS|
 In order to invoke |Office iOS| when opening an Office file from your app, you must use the URL schemes that Word,
 Excel, and PowerPoint for iOS register when they are installed.
 
-General information on the Office URL schemes can be found here:
-https://msdn.microsoft.com/en-us/library/office/dn906146.aspx#sectionSection1
+URL Schemes for Inovking Office
+-------------------------------
 
-In order to open files on your WOPI host, you must do the following:
+The following is the list of scheme names used to invoke Office:
 
-* Pass the :term:`WOPISrc` to the file via the URL schemes
-* Along with the WOPISrc of the file, you must include:
+ * ms-word:
+ * ms-powerpoint:
+ * ms-excel:
 
-  * The service identifier, denoted by the ``|d|`` parameter
-  * The user identifier, denoted by the ``|e|`` parameter
-  * The file name, with extension, denoted by the ``|n|`` parameter
+The following informaiton must be included along with the URL scheme: 
 
+* The mode to open the file in. Values values are:
+ * ``ofv`` for opening as read-only
```

Comment:
Add two spaces for nested lists. The `*` should align with the text of the parent bullet.